src/index.ts

```diff
@@ -82,3 +85,28 @@ export const parseKind = (kind: string, { rootSeparator, groupSeparator }: Separ
     groups,
   };
 };
+
+/**
+ * csf story metadata attached to the story export function
+ */
+export interface StoryMetadata {
+  story?: {
+    /**
+     * story name if differnet from the export name
+     */
+    name?: string;
+
+    /**
+     * optional collection of properties, which values
+     * will be passed onto the story function
+     */
+    properties?: StoryProperties;
+
+    /**
+     * optional collection of story parameters
+     */
+    parameters?: {
+      [key: string]: any;
```

Comment:
```suggestion
      [key: string]: Object<string, any>;
```

(Apologies for my made up typescript).

I'm not sure it's 100% stuck to but the convention at least is that parameters should always be an object.

Reply:
@tmeasday for reference here is the parameters code from lib/api:
```
export interface Parameters {
  fileName?: string;
  options?: OptionsParameter;
  [key: string]: any;
}
```